include/aspect/material_model/rheology/peierls_creep.h

```diff
@@ -50,10 +50,7 @@ namespace aspect
         double glide_parameter_p;
         double glide_parameter_q;
         double fitting_parameter;
-        double stress_c;
-        double a_c;
-        double b_c;
-        double d_ref;
+        double stress_coff;
```

Comment:
Can you call this `stress_cutoff`? Otherwise it could be mistaken for `stress coefficient`.
Ditto for all the other "coffs".